classes/rust.bbclass

```diff
@@ -86,6 +86,21 @@ HOST_CPPFLAGS ?= "${CPPFLAGS}"
 
 EXTRA_OECONF_remove = "--disable-static"
 
+do_rust_bin_fixups() {
```

Comment:
So the goal here is to set the rpath to the proper rustlib path? I thought some handling landed in rust to automagically set the proper rpaths during build time and installation time. @jmesmon is my understanding of this wrong? This comment shouldn't block merging this series in but more of a clarification.